infra/MAINTAINERS

```diff
@@ -0,0 +1,6 @@
+Adam Korcz <adam@adalogics.com> @AdamKorcz
```

Comment:
nit: I think it woiuld be better to use a real format, like CSV. What if someone mistakenly adds maintainer with a middle name.

Reply:
Adding a middle name is fine. But yeah, CSV is a better format, I will update it.